__tests__/response/flushHeaders.test.js

```diff
@@ -94,31 +96,25 @@ describe('ctx.flushHeaders()', () => {
       })
     })
 
-    app.listen(async function (err) {
-      if (err) return done(err)
-
-      const server = this
-      const port = this.address().port
-
-      http.request({
-        port
-      })
-        .on('response', res => {
-          const onData = () => done(new Error('boom'))
-          res.on('data', onData)
-
-          // shouldn't receive any data for a while
```

Comment:
this test is not valid since the listener is added and removed and the test is complete on the same tick - the logic only happens on next tick. we can keep it for now but it's a bad test

Reply:
I updated this test! 👀